nbs_tests/test_thread.ipynb

```diff
@@ -34,7 +34,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from multiprocessing.pool import ThreadPool
</pre><p>just a test what happens to these comments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MannLabs/alphabase/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>